test/known_problems/should_pass/list_nth.erl

```diff
@@ -0,0 +1,32 @@
+-module(list_tail).
+
+-compile([export_all]).
+
+% Pass
+-spec nth1([integer()]) -> integer().
+nth1([]) -> 0;
+nth1(Xs) -> lists:nth(1, Xs).
```

Comment:
Good point. I assumed they would indeed and was surprised they didn't. If something gets "accidently" fixed, I wish it would tell us.

Reply:
It would be possible to have. https://github.com/josefs/Gradualizer/blob/f4e95cb1baf38300e1838d5663a49986232a766b/test/test.erl#L55-L58 could take inspiration from https://github.com/josefs/Gradualizer/blob/f4e95cb1baf38300e1838d5663a49986232a766b/test/test.erl#L37-L49